modules/study/src/test/CommentParserTest.scala

```diff
@@ -67,10 +67,10 @@ class CommentParserTest extends LilaTest:
     assertEquals(C("Hello there [%emt 2:10] something else").emt, Some(Centis(780000)))
 
   test("parse clock: alt format"):
-    assertEquals(C("Hello there [%clk 2:10.33] something else").clock, Some(Centis(783300)))
+    assertEquals(C("Hello there [%clk 2:10.33] something else").clock, Some(Centis(781980)))
```

Comment:
looks like a test change that is validating a BC break?

`2*3600 + 10*60 + 33 == 7833`, what is `7819.8`?